DATA_FORMAT.md

```diff
@@ -0,0 +1,77 @@
+# JSON Data fields
+
+## Message Data
+These fields are the primary data fields containing the most basic message data and used to identify the specific device. 
+For some devices these are the *only* fields contained in the message, as the message itself consitutes an event from this
+particular device model.
+
+* **time** (string) (Required)
+  * Time stamp. String containing date and time of when the message was received. Format is dependent on
+    current locale unless the `-U` command line argument is used.
+
+* **model** (string) (Required)
+  * Device model. Human readable string describing the model consisting of manufacturer name 
+    and manufacturers model description
+
+* **id** (string) (Optional)
+  * Device identification. Used to differentiate between devices of same *model*. 
+    Depending of device model it may be a non-volatile value programmed into the device, 
+    a volatile value that changes at each power on (or battery change), or a value configurable by
+    user e.g. by switch or jumpers. No assumptions should be made to the id value other than it contains
+    a unique sequence of alphanumeric characters.
+
+* **channel** (string) (Optional)
+  * Secondary device identification. For devices with more than one identification value 
+    (e.g. both an internal value and a switch).
+
+* **mic** (string) (Optional)
```

Comment:
In terms of a data architecture, MIC as currently defined is redundant data.  

-- It will never change between messages on the same device.  So there is no point in sending and storing it *on every message*.
-  It isn't an attribute of the message., It's an attribute of *the device/the decoder*.

The use case of disabling devices that don't meet the specified level seems like a runtime option for rtl_433. 

Alternatively rtl_433 should output (on start up) a machine readable message about what devices are enabled, and their attributes like MIC.

Reply:
The idea is that the consumer can add more trust to a message if mic is present. I get your point but in a blind scanning scenario when you pickup unknown transmitters it is much more user friendly to directly see that the protocol has an integrity check.